packages/components/carousel/src/Carousel.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { Component, MouseEventHandler } from 'react';
+import { Carousel as ResponsiveCarousel } from 'react-responsive-carousel';
+import 'react-responsive-carousel/lib/styles/carousel.min.css';
+import type { CarouselProps, Content } from './carousel.types';
+import { CarouselStyled, CarouselHtmlStyled } from './carousel.styles';
+
+class Carousel extends Component<CarouselProps> {
+  renderContent(content: Content): React.ReactNode {
+    if ('src' in content && 'alt' in content) {
```

Comment:
You don't need to specify what kind of content you need to receive. Each item carousel must have only children!!